pkg/encryption/cipher.go

```diff
@@ -59,6 +80,17 @@ func SignedValue(seed string, key string, value string, now time.Time) string {
 }
 
 func cookieSignature(args ...string) string {
+	h := hmac.New(sha256.New, []byte(args[0]))
+	for _, arg := range args[1:] {
+		h.Write([]byte(arg))
+	}
+	var b []byte
+	b = h.Sum(b)
+	return base64.URLEncoding.EncodeToString(b)
+}
```

Comment:
This is basically the same as the content of LegacyCookieSignature, how about extracting to a helper somehow?

```
func cookieSignature(args ...string) string {
  return cookieSignatureFrom(sha256.New, args...)
}

func legacyCookieSignature(args ...string) string {
  return cookieSignatureFrom(sha1.New, args...)
}
```